mentors/Makefile

```diff
@@ -108,14 +109,14 @@ shell: docker-image-exists $(WATSON_CREDENTIALS)
 # Complete build of mentor data and videos
 # Runs build if necessary
 # Generates data files
-%/videos:
-	$(MAKE) $*/videos/update
+videos/%:
+	$(MAKE) update/$*/videos
 
 # Complete build of mentor data and videos
 # Runs build if necessary
 # Generates data files
-.PHONY: %/videos/update
-%/videos/update: %/build docker-image-exists
+.PHONY: update/%/videos
+update/%/videos: %/build docker-image-exists
```

Comment:
I think the other solution for the loop issue you encountered would be to add the directory creation also to the `videos/%` rule above, e.g.

```
videos/%:
	mkdir -p $(shell pwd)/$*/data
	$(MAKE) **update/$*/videos
```

...also possibly related: the update rule doesn't immediately create that videos folder either?